scripts/check-test-generate.sh

```diff
@@ -31,12 +28,16 @@ variables:
 # Cached directories shared between jobs & pipelines per-branch per-runner
 cache:
   key: $CI_COMMIT_REF_SLUG
+  when: 'always'
   paths:
     - ./tmp/npm/
     - ./tmp/ts-node-cache/
     # `jest` cache is configured in jest.config.js
     - ./tmp/jest/
```

Comment:
Should we keep the cache the same across the board?

In `check-test-generate.sh` there's no `Homebrew` and `chocolatey`. But in `build-platforms-generate.sh` there is.

So then either we put it everywhere to simplify things, or optimise per pipeline. I'd prefer to do it everywhere to simplify things.

Reply:
It should be fine to have it everywhere. All it means is that at the end of the script there's a warning for caches that it couldn't find any files for, but the pipeline still succeeds even with the warning.

Reply:
Yep, that's fine, let's add it everywhere.